app/ide-desktop/lib/dashboard/src/pages/subscribe/Subscribe/components/Card.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * @file
+ *
+ * Card component
+ */
+import * as React from 'react'
+
+import * as tw from 'tailwind-merge'
+
+import type * as text from '#/text'
+
+import * as textProvider from '#/providers/TextProvider'
+
+import * as aria from '#/components/aria'
+
+/**
+ * Card props
+ */
+export interface CardProps extends React.PropsWithChildren {
+  /**
+   * Card title
+   */
+  readonly title: text.TextId
+  /**
+   * Card subtitle
+   */
+  readonly subtitle: text.TextId
+  /**
+   * Card features
+   */
+  readonly features: string[]
+  readonly pricing?: text.TextId
+  readonly submitButton?: React.ReactNode
+  readonly learnMore?: React.ReactNode
+  readonly className?: string
+}
+
+/**
+ * Card component
+ */
+export function Card(props: CardProps) {
+  const { children, features, submitButton, title, subtitle, pricing, learnMore, className } = props
+
+  const { getText } = textProvider.useText()
+
+  return (
+    <div className={tw.twMerge('rounded-xl border border-stone-300 p-4', className)}>
+      <aria.Heading className="text-3xl leading-10" level={2}>
```

Comment:
It's better to define a component for that instead of defining a `className`. Period
I've created a dedicated `Text` components in another PR `Billing page`